src/Components/TopicMenu.jsx

```diff
@@ -0,0 +1,28 @@
+import { useState, useEffect } from "react";
+import { getTopics } from "../api";
+
+export default function TopicMenu() {
+  const [topics, setTopics] = useState([]);
+
+  useEffect(() => {
+    getTopics().then((topicList) => {
+      setTopics(topicList.topics);
+    });
+  }, [setTopics]);
```

Comment:
I'm not sure what's going on here but you shouldn't need to have a setState function in your useEffect dependancies